src/aind_data_schema_models/registry.py

```diff
@@ -20,6 +20,13 @@ class Addgene(_Registry):
     abbreviation: Literal["ADDGENE"] = "ADDGENE"
 
 
+class EMAPA(_Registry):
```

Comment:
let's have the class name be EdinburghMouseAtlasProject to match the other registries.